playbooks/openshift-node/private/join.yml

```diff
@@ -36,54 +62,46 @@
 
   - name: Distribute bootstrap kubeconfig if one does not exist
     copy:
-      content: "{{ bootstrap_credentials.content | b64decode }}"
+      src: "{{ hostvars['localhost'].mktemp.stdout }}/bootstrap.kubeconfig"
```

Comment:
Do we still need to slurp the config above?  What was wrong with the slurping approach btw?  That looked pretty efficient.

Reply:
Right, I'm thinking to revert this part. Temp dir is still needed for CNI configs though